src/mono/wasm/Wasm.Build.Tests/ProjectProviderBase.cs

```diff
@@ -282,7 +282,7 @@ public static string FindSubDirIgnoringCase(string parentDir, string dirName)
         List<string> files = new()
         {
             Path.Combine(paths.BinDir, "publish", $"{buildArgs.ProjectName}.dll"),
-            Path.Combine(paths.ObjWasmDir, "driver.o"),
```

Comment:
We shouldn't drop this. If this no longer depends on assemblies changing then the tests can expect that now, and expect `runtime.o` to change when assemblies change.